utils/wasm-builder/src/cargo_toolchain.rs

```diff
@@ -66,7 +66,7 @@ impl Toolchain {
         static TOOLCHAIN_CHANNEL_RE: Lazy<Regex> = Lazy::new(|| {
             // This regex is borrowed from the rustup code and modified (added non-capturing groups)
```

Comment:
Probably you have to adjust this comment as well as it is no longer a borrowed pattern. Unless they have updated it in rustup as well ;-)

Reply:
done